app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +393,47 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div ref={menuRef} style={{ display: "flex", flexDirection: "row" }}>
+            <div
+              style={{ marginRight: 16, display: "flex", alignItems: "center" }}
+            >
+              <ReportButton />
+            </div>
+            <div
+              className={classes.menuBtn}
```

Comment:
This should be implemented with the `button` element as using a `div` for buttons with an `onClick` is bad for accessibility. An `IconButton` from MUI might be a good choice here to use instead.

Reply:
I added a Button, should i create a custom Button to which i add the icon or should i leave it as <Button ...> ?

Reply:
Not sure what you mean.

Darren is talking about `<button>`, the native html element. We also have a button component we use, `<Button>`, which has some extra stuff added to the MUI component `<Button>`. However I'm not sure it will work - if it doesn't, you could use MUI's `<ButtonBase>` component, which is more customizable.